extensions/podman/vite.config.js

```diff
@@ -45,7 +45,7 @@ const config = {
       formats: ['cjs'],
     },
     rollupOptions: {
-      external: ['@podman-desktop/api', ...builtinModules.flatMap(p => [p, `node:${p}`])],
+      external: ['@podman-desktop/api', 'ssh2', ...builtinModules.flatMap(p => [p, `node:${p}`])],
```

Comment:
Shouldn't it be explicitly included into dist for podman ext as node_modules/ssh2? I tried to to declare packages from podman-desktop root package.json as external for extension and they would not get resolved at runtime when extension loading.

Also podman ext declares dependency on ssh2@1.15.0, but podman desktop includes 1.11.0 version in app.asar file.

Reply:
there is no need as it's using the dependency from podman desktop

All versions of ssh2 work as well as there is no usage of fancy stuff

Reply:
I see now, it works for built-in extensions, because of the way they are included into asar file.

![image](https://github.com/user-attachments/assets/7fe686dc-a4fb-408f-8a39-2dd553570bc6)